src/commands/Math.js

```diff
@@ -0,0 +1,142 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+const math = require('mathjs');
+//
+
+function hasTypes(i) {
+	return /[+-/*^%]/.test(i);
+};	
+async function request(i) {
+   	return await superagent.get(`http://numbersapi.com/${i}?json`);
+};  
+//
+
+class Math extends Command {
+	constructor(...args) {
+		super(...args);
+		
+		this.aliases      = ['math'];
+		this.module       = 'Fun';
+		this.description  = 'Math command';
+		this.usage        = 'math [math expression]';
+		this.example      = 'math 152 + 37';
```

Comment:
Would be cool to add more examples like the ones in mathjs website, and specially the `math fact` example